assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -298,6 +277,10 @@ function MediaPane(props) {
     [hasMore, isMediaLoading, isMediaLoaded, setNextPage]
   );
 
+  const onSearch = (v) => setSearchTerm({ searchTerm: v });
+
+  const autoSearchDebounceMedia3p = useFeature('autoSearchDebounceMedia3p');
```

Comment:
A bit confusing to use a media3p flag for local media

Reply:
Renamed the flag.